backend/migrations/README.md

```diff
@@ -0,0 +1,41 @@
+Single-database configuration for Flask.
```

Comment:
Perfect! Thanks for updating the doc!

BTW github can render md (markdown) files and this is how it looks like:
https://github.com/meetingbook/meetingbook/blob/25b13de22a29da851831f6cc8e68c64ca7b8d1e7/backend/migrations/README.md

Reply:
![image](https://user-images.githubusercontent.com/83371532/130658445-c6da5f8f-bace-4629-ad82-5630a4e56d6c.png)